lib/reek/configuration/configuration_file_finder.rb

```diff
@@ -58,11 +58,11 @@ def load_from_file(path)
 
           begin
             configuration = YAML.load_file(path) || {}
+            SchemaValidator.new(configuration).validate
```

Comment:
Moving it here so we can catch configuration file errors raised by the validator.
The benefit is that we have access to the config file path here.

If this is not ideal, I can revert to the way we had but won't have access to the config file path.

Reply:
I think this makes sense :+1: